portfolio/src/main/java/com/google/sps/servlets/InterestsServlet.java

```diff
@@ -1,18 +1,28 @@
 package com.google.sps.servlets;
 
 import java.io.IOException;
+import java.util.*;
```

Comment:
generally, it's better to import individual modules rather than the whole package to avoid importing unnecessary modules and avoid name clashes, heres more info if you're interested: https://medium.com/@aayushsharda/why-is-it-better-to-avoid-in-import-statements-in-java-ff9ceaca8aeb